src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,94 @@
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { useEffect, useMemo, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useSearchParams } from 'react-router-dom';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const [searchParams] = useSearchParams();
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople)
+      .catch(() => setHasError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const filteredPeople = useMemo(() => {
+    let filtered = people;
+
+    const query = searchParams.get('query')?.toLowerCase() || '';
+    const centuries = searchParams.getAll('centuries');
+    const sort = searchParams.get('sort');
+    const order = searchParams.get('order') === 'desc' ? -1 : 1;
+
+    if (query) {
+      filtered = filtered.filter(
+        person =>
+          person.name.toLowerCase().includes(query) ||
+          person.motherName?.toLowerCase().includes(query) ||
+          person.fatherName?.toLowerCase().includes(query),
+      );
+    }
+
+    if (centuries.length > 0) {
+      filtered = filtered.filter(person =>
+        centuries.includes(Math.ceil(person.born / 100).toString()),
+      );
+    }
+
+    if (sort) {
+      filtered = [...filtered].sort((a, b) => {
+        const aValue = a[sort as keyof Person] || '';
+        const bValue = b[sort as keyof Person] || '';
+
+        return aValue > bValue ? order : aValue < bValue ? -order : 0;
```

Comment:
The sorting logic compares `aValue` and `bValue` using the `>` and `<` operators. This works well for strings and numbers but could lead to unexpected results if the values are of different types or if they include `null` or `undefined`. Consider adding type checks or default values to ensure consistent sorting behavior.